homework/src/index.js

```diff
@@ -1,47 +1,91 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
-      }
-    };
-    xhr.onerror = () => cb(new Error('Network request failed'));
-    xhr.send();
-  }
-
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach((key) => {
-      const value = options[key];
-      if (key === 'html') {
-        elem.innerHTML = value;
-      } else {
-        elem.setAttribute(key, value);
-      }
-    });
-    return elem;
-  }
-
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
-      }
-    });
-  }
-
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
-
-  window.onload = () => main(HYF_REPOS_URL);
-}
+    function fetchJSON(url, cb) {
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', url);
+        xhr.responseType = 'json';
+        xhr.onload = () => {
+            if (xhr.status < 400) {
+                cb(null, xhr.response);
+            } else {
+                cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+            }
+        };
+        xhr.onerror = () => cb(new Error('Network request failed'));
+        xhr.send();
+    }
+
+    function createAndAppend(name, parent, options = {}) {
+        const elem = document.createElement(name);
+        parent.appendChild(elem);
+        Object.keys(options).forEach((key) => {
+            const value = options[key];
+            if (key === 'text') {
+                elem.innerHTML = value;
+            } else {
+                elem.setAttribute(key, value);
+            }
+        });
+        return elem;
+    }
+
+    function main(url) {
+
+
+        const root = document.getElementById('root');
+
+        const div = createAndAppend('div', root, { class: 'container' });
+        const header = createAndAppend('h3', div, { text: "HYF Repositories  ", class: 'header' });
+        const select = createAndAppend('select', header, { class: "select" });
+        const table = createAndAppend('table', root, { class: 'table' });
+        const tbody = createAndAppend('tbody', table);
+        const tr = createAndAppend('tr', tbody);
+        createAndAppend('td', tbody, { text: "Repository: ", class: 'label' });
+        const repoTitle = createAndAppend('a', tbody);
```

Comment:
The `a` element should be a child of the `td` element, not of the `tbody` element. For applications like this it is always a good idea to use the Chrome inspector to look at the HTML your app generated. Because I'm sure that if you were to write this HTML by hand rather than through JS you would do it correctly.